.circleci/config.yml

```diff
@@ -43,6 +53,28 @@ commands:
             sudo apt-get update -y --quiet
             sudo apt-get install -y --quiet git build-essential google-cloud-sdk shellcheck zsh
 
+  install_machine:
+    steps:
+      - run:
+          name: install VM dependencies (fuse+recent golang)
+          environment:
+            GOROOT: /usr/local/go
+            GOPATH: /go
+          command: |
+            GO_VERSION=1.13.5
+            CI_USER=$(id -u)
+            sudo apt-get install --quiet fuse
+
+            cd /tmp
+            wget --no-verbose https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz
```

Comment:
does the FUSE CI setup play without this golang environment install?

not saying we need to change the method of installing golang.  wondering if the golang and fuse installs are conceptually grouped, or if this patch is more about going with what works in order to prop up fuse on ci in the first place.

Reply:
@ransomw1c yes I've added to golang to run the unit tests only (since I spin up a VM, I did not go so far so as to use that VM _only_ to build docker containers to run golang. That could have been a way to go, albeit a lengthy one.

We also might want to use the VM as  a more general purpose docker server specifically to run privileged containers (which can't be achieved with the standard circleCI docker), as per required by the fuse driver, but this is out of the scope of this PR: going further toward CI optimization, one could pre-built the VM image and make it available globally to our circleCI account etc - as for now, I don't have the grants to do something like that.